### PR TITLE
chore: migrate to bbolt instead of sql-lite

### PR DESCRIPTION
**Pull request by bryancua**
Remove sqlite dependency (due to it requiring CGO and therefore not allowing cross compile) into another file database that doesn't require CGO.

### CODE REVIEW

File: handler-host.go [hunk 1]
```diff
@@ -2,16 +2,17 @@ package logx
...
 	"errors"
 	"fmt"
 	"io"
-	"strings"
 	"sync"
 	"time"
 
-	_ "github.com/mattn/go-sqlite3"
+	"github.com/etcd-io/bbolt"
+	uuid "github.com/nu7hatch/gouuid"
```
Comment (tmathews) [inline handler-host.go:13]: We should unifi which lib we use across projects. I know we using anotherone somewhere.

File: handler-host.go [hunk 2]
```diff
@@ -401,10 +418,32 @@ func (h *HostHandler) ReadResponses(conn *tls.Conn, errCh chan error) {
...
+			return nil
+		}
+		return b.Delete([]byte(id))
+	})
+}
+
+func (h *HostHandler) GetLocalLogs() (arr []*BaseHostLog, err error) {
+	arr = make([]*BaseHostLog, 0, 10)
+	err = h.db.View(func(tx *bbolt.Tx) error {
+		b := tx.Bucket(BucketName)
```
Comment (tmathews) [inline handler-host.go:433]: This chunk is kinda repetitive.

**Review by tmathews — APPROVED** [review]

**Comment by tmathews** [comment]
NoSQL is ugly as heck.

**Comment by tmathews** [comment]
Really not a fan of the JS-esque style db calling. Don't think I'll be using this library in the future.

**Comment by tmathews** [comment]
PR lacks description, negative points.

**Comment by bryancua** [comment]
> NoSQL is ugly as heck.

Yes it is. Lots of converting to byte arrays

**Comment by bryancua** [comment]
> PR lacks description, negative points.

LOL. Thought the title was clear enough

**Comment by bryancua** [comment]
> Really not a fan of the JS-esque style db calling. Don't think I'll be using this library in the future.

You mean the Mongo style.

File: handler-host.go (hunk 1 above)
Comment (bryancua) [reply handler-host.go:13]: indeed

File: handler-host.go (hunk 2 above)
Comment (bryancua) [reply handler-host.go:433]: The JSON type is different.

**Pull request merged**